scripts/generateProperties.js

```diff
@@ -27,8 +27,7 @@ const outFile = fs.createWriteStream(
     encoding: "utf-8"
   }
 );
-const dateToday = new Date();
-const [dateTodayFormatted] = dateToday.toISOString().split("T");
+const [dateTodayFormatted] = new Date().toISOString().split("T");
```

Comment:
Nit: I find this part of JavaScript very confusing so I prefer parentheses.

```suggestion
const [dateTodayFormatted] = (new Date()).toISOString().split("T");
```